src/templates/components/contact.html

```diff
@@ -18,14 +18,11 @@ <h3>Contact Us</h3>
       </p>
     {% else %}
       <div id="noid-container">
-        <p>If you have a WCA account, please <a href="/login">log in</a></p>
+        <p>In order to contact us, you will need to login in with your WCA account <a href="/login">here</a>.</p>
```

Comment:
I'd prefer to not make it required to use a WCA account, as this will deter other people like media from contacting us.  Can you revert to the old wording?

Reply:
Ah my mistake, I misinterpreted the meaning of this. Fixed!